content/en/docs/releases/supported-releases/index.md

```diff
@@ -78,9 +79,22 @@ Please keep up-to-date and use a supported version.
 {{< /warning >}}
 
 | Minor Releases   | Patched versions with no known CVEs                  |
-|------------------|------------------------------------------------------|
+| ---------------- | ---------------------------------------------------- |
 | 1.16.x           | 1.16.0+                                              |
 | 1.15.x           | 1.15.3+                                              |
 | 1.14.x           | 1.14.5 - End of life. A new CVE will NOT be patched. |
 | 1.13.x           | 1.13.9 - End of life. A new CVE will NOT be patched. |
 | 1.12 and earlier | None, all versions have known vulnerabilities.       |
+
+## Supported Envoy Versions
+
+ Istio's data plane is based on [Envoy](https://github.com/envoyproxy/envoy).
+
+ The relationship between the two project's versions:
+
+ | Istio version | Envoy version |
+ | ------------- | ------------- |
+ | 1.16.x        | 1.24.x        |
+ | 1.15.x        | 1.23.x        |
+
+ In general, Istio releases tend to map one to one with Envoy releases. You can find the precise Envoy commit used by Istio in [istio/proxy](https://github.com/istio/proxy/blob/master/WORKSPACE#L38).
```

Comment:
istio here is what made it to be added to .spelling, looks like

Reply:
Yep. Looking for a fix.

Reply:
We don't want istio in .spelling else anyone can add istio in a Doc and it won't get flagged. Typically we add backticks for specific items.